src/en/charms-storage.md

```diff
@@ -304,6 +305,15 @@ config:
   security.privileged: "true"
 ```
 
+### Remove storage
+
+Specify one or more unit storage, volume, or filesystem IDs, which can be found
+in the output of `juju storage`, like this:
+
+```bash
+juju remove-storage pgdata/0
+```
+
```

Comment:
Paging @axw. When you have a few minutes, can you help me answer the questions @evilnick poses above? I think anticipating and answering these up front in our docs will help our users.